ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -5539,6 +5539,7 @@ static int set_sec_mod_handler(ldmsd_req_ctxt_t reqc)
 		if (isdigit(value[0])) {
```

Comment:
@nichamon the value variable here is leaked throughout this function. Although it is freed at the end, because it is reused throughout, only the last value is actually freed. This fix is part of the "memory-leak" branch I was going to send you, but I think my change and yours will conflict. Maybe I just send you the branch and you combine them all into one pull request that you submit?

Reply:
I'll do that.